react/src/components/UserCredentialList.tsx

```diff
@@ -0,0 +1,518 @@
+            title: t('credential.KeyAge'),
+            dataIndex: 'created_at',
+            render: (createdAt) => {
+              return `${dayjs().diff(createdAt, 'day')}${t('credential.Days')}`;
+            },
+            sorter: true,
+          },
+          {
+            key: 'createdAt',
+            title: t('credential.CreatedAt'),
+            dataIndex: 'created_at',
+            render: (createdAt) => dayjs(createdAt).format('lll'),
+            sorter: true,
+            defaultSortOrder: 'descend',
+          },
+          {
+            key: 'resourcePolicy',
+            title: t('credential.ResourcePolicy'),
+            dataIndex: 'resource_policy',
+            sorter: true,
+          },
+          {
+            key: 'allocation',
+            title: t('credential.Allocation'),
+            render: (record) => {
+              return (
+                <Flex direction="column" align="start">
+                  <Typography.Text>
+                    {record.concurrency_used} / {record.concurrency_limit}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.Sessions')}
+                    </Typography.Text>
+                  </Typography.Text>
+                  <Typography.Text
+                    style={{
+                      fontSize: token.fontSizeSM,
+                    }}
+                  >
+                    {record.rate_limit}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.ReqPer15Min')}
+                    </Typography.Text>
+                  </Typography.Text>
+                  <Typography.Text
+                    style={{
+                      fontSize: token.fontSizeSM,
+                    }}
+                  >
+                    {record.num_queries}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.Queries')}
+                    </Typography.Text>
+                  </Typography.Text>
+                </Flex>
+              );
+            },
+          },
+          {
+            key: 'control',
+            title: t('general.Control'),
+            fixed: 'right',
+            render: (record) => {
+              return (
+                <Flex gap={token.marginXS}>
+                  <Button
+                    type="link"
```

Comment:
Please use "text" type instead of "link" and apply the success color to `InfoCircleOutlined` to match the colors with the user list.